modules/distr-tracing-otel-troubleshoot-logging-exporter.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * /distr_tracing/distr_tracing_otel/distr-tracing-otel-troubleshooting.adoc
+:_content-type: PROCEDURE
+[id="distr-tracing-otel-troubleshoot-logging-exporter_{context}"]
+= Logging exporter
```

Comment:
Nit: I think this would benefit from a more descriptive title, such as "Exporting collected data using the logging exporter".